### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by anbraten**
# changes
- change default log level from `warn` to `info`
- add log info message at server / agent start with version info
- cleanup server startup function
- change some level of some log calls from `info` to `debug`

### CODE REVIEW

File: cmd/agent/agent.go [hunk 1]
```diff
@@ -69,7 +70,7 @@ func loop(c *cli.Context) error {
 		)
 	}
 
-	zerolog.SetGlobalLevel(zerolog.WarnLevel)
+	zerolog.SetGlobalLevel(zerolog.InfoLevel)
```
Comment (lafriks) [inline cmd/agent/agent.go:73]: Ideally this should be settable by env variable

Comment (anbraten) [reply cmd/agent/agent.go:73]: You can set it. This is the value it uses in case you haven't specified it.

File: cmd/agent/agent.go [hunk 2]
```diff
@@ -69,7 +70,7 @@ func loop(c *cli.Context) error {
...
 	if zerolog.GlobalLevel() <= zerolog.DebugLevel {
```
Comment (6543) [inline cmd/agent/agent.go:74]: shouldn't this be after line 76 ?

Comment (anbraten) [reply cmd/agent/agent.go:74]: Good point that's pretty strange 🤔 Line 72-75 should be in general behind 85?

Comment (anbraten) [reply cmd/agent/agent.go]: changed it

**Review by 6543 — APPROVED** [review]

**Pull request merged**